DQMOffline/Trigger/src/HLTMuonMatchAndPlot.cc

```diff
@@ -484,6 +492,35 @@ void HLTMuonMatchAndPlot::analyze(Handle<MuonCollection>   & allMuons,
     }
   }
 
+  string nonSameSignPath = hltPath_;
+  bool ssPath=false;
+  if ( nonSameSignPath.rfind("_SameSign") < nonSameSignPath.length()){
+    ssPath = true;
+    nonSameSignPath = boost::replace_all_copy<string>(nonSameSignPath, "_SameSign","");
+    nonSameSignPath = nonSameSignPath.substr(0, nonSameSignPath.rfind("_v")+2);
+  }
+  bool passTriggerSS = false;
+  if (ssPath){
+    for ( unsigned int hltIndex = 0; hltIndex < numTriggers; ++hltIndex){
+      passTriggerSS = passTriggerSS ||  (trigNames.triggerName(hltIndex).find(nonSameSignPath) != std::string::npos && triggerResults->wasrun(hltIndex) && triggerResults->accept(hltIndex));
```

Comment:
hi @sscruz - does this string comparison need to search the entire string or just the beginning of it to match triggerName(hltIndex) to nonSameSignPath?

Reply:
hi @davidlange6  ,  no, we just need to match the beginning

Reply:
hi @sscruz - sorry to answer slowly - in that case you can do a string comparison instead of a find:

trigNames.triggerName(hltIndex).substr(0,nonSameSignPath.size()) == nonSameSignPath

Reply:
sure! thanks a lot!